### DISCUSSION TITLE
How to use the Power Button feature?

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#176] [Fixed Issues]
Hello, 

How to use the power button feature ?

Thanks for help

**Comment by findnix** [comment]
To extend the question:
_... in combination with an ArgonONE case?_

At the moment I use this: https://github.com/Elrondo46/argonone, which also allows temperature controlled fans speed. 
Woks fine, but has an unnecessarily large footprint, because it brings its own pyhton3 environment with it. 
```
rAudio:~  #  systemctl status argononed
* argononed.service - Argon One Fan and Button Service
     Loaded: loaded (/usr/lib/systemd/system/argononed.service; enabled; vendor preset: disabled)
     Active: active (running) since Sun 2021-02-07 14:04:45 CET; 11min ago
   Main PID: 907 (python3)
      Tasks: 4 (limit: 4915)
     CGroup: /system.slice/argononed.service
             `-907 /opt/argonone/bin/python3 /opt/argonone/bin/argononed.py

Feb 07 14:11:37 rAudio python3[907]: Temp: 45.2 FanSpeed: 0
Feb 07 14:12:07 rAudio python3[907]: Temp: 46.7 FanSpeed: 0
Feb 07 14:12:37 rAudio python3[907]: Temp: 46.2 FanSpeed: 0
Feb 07 14:13:07 rAudio python3[907]: Temp: 45.7 FanSpeed: 0
Feb 07 14:13:37 rAudio python3[907]: Temp: 46.2 FanSpeed: 0
Feb 07 14:14:07 rAudio python3[907]: Temp: 46.2 FanSpeed: 0
Feb 07 14:14:37 rAudio python3[907]: Temp: 45.7 FanSpeed: 0
Feb 07 14:15:07 rAudio python3[907]: Temp: 45.7 FanSpeed: 0
Feb 07 14:15:37 rAudio python3[907]: Temp: 46.7 FanSpeed: 0
Feb 07 14:16:07 rAudio python3[907]: Temp: 45.2 FanSpeed: 0

```

**Comment by rern** [comment]
**Power button** without extra circuit/components

_System > GPIO Devices > Power Button:_
- Momentary switch - DPST / DPDT (2 contacts)
	- ON: Connect between fixed pin `5` and ground.
	- OFF: Connect between selectable pin and ground.
- LED (optional)
	- Connect between selectable pin and ground.
		- On when boot finished.
		- 12V integrated LED push button can be used.
		- Caution: 100-200 ohms resister needed for LED current limiting.
